client/scss/components/_dropdown-button.scss

```diff
@@ -0,0 +1,136 @@
+// stylelint-disable selector-attribute-name-disallowed-list
+$separator: 1px solid theme('colors.white-15');
+$radius: theme('borderRadius.sm');
+
+.w-dropdown-button {
+  // Cosmetic details based on live tooltip placement,
+  // implemented CSS-only by using Tippy.js `data-placement` and `:has`.
+  --primary-button-radius-top: #{$radius};
+  --primary-button-radius-bottom: #{$radius};
+  --toggle-button-radius-top: #{$radius};
+  --toggle-button-radius-bottom: #{$radius};
+  --first-item-border-top: 0;
+  --last-item-border-top: 0;
+  // Set each separately so a button that’s both first and last is correct.
+  --first-item-start-start-radius: 0;
+  --first-item-start-end-radius: 0;
+  --last-item-end-start-radius: 0;
+  --last-item-end-end-radius: 0;
+
+  display: flex;
+  // Make sure the tooltip within will match this element’s width.
+  position: relative;
+
+  [data-tippy-root] {
+    // Make sure the tooltip within will match this element’s width.
+    width: 100%;
+  }
+
+  @supports not selector(:has(*)) {
```

Comment:
As of now, Firefox 115 ESR is the only browser we support which doesn’t have `:has` support.